src/annotator/integrations/pdf.js

```diff
@@ -59,6 +78,12 @@ export class PDFIntegration {
       subtree: true,
     });
 
+    /**
+     * Amount of time to wait for re-anchoring to complete when scrolling to
+     * an anchor in a not-yet-rendered page.
+     */
+    this._reanchoringWait = options.reanchoringWait ?? 3000;
```

Comment:
I had to go look at the implementation of the `_waitForAnnotationToBeAnchored` method to understand that this is a max wait/timeout value. Perhaps the comment or the attribute name could be adjusted to make this more clear? I initially read this as a "wait exactly this amount of time."